packages/resizable/src/useResizable/useResizable.spec.tsx

```diff
@@ -270,24 +269,30 @@ describe('useResizable', () => {
       }),
     );
 
-    // Focus the resizer
-    const resizerProps = result.current.getResizerProps();
-    act(() => {
-      resizerProps?.onFocus();
-    });
+    // Get resizer props
+    let resizerProps = result.current.getResizerProps();
 
-    // Press right arrow to increase size
+    // Trigger keyDown directly since the hook handles keyboard events directly
     act(() => {
-      fireEvent.keyDown(window, { code: keyMap.ArrowRight });
+      resizerProps?.onKeyDown({
+        code: 'ArrowRight',
+        preventDefault: jest.fn(),
+      } as unknown as React.KeyboardEvent);
     });
```

Comment:
Was initially confused by this. We're testing that the handler triggers the resize callback, right? 
Feels strange since this _feels_ like we should be using `userEvent`, but that would technically be testing an implementation detail (did the consumer pass `onKeyDown` to the right element)

Reply:
Yeah, this is correct. It's testing that it's calling the resize callback and that the correct size is also returned.